gptel-bedrock.el

```diff
@@ -635,7 +642,9 @@ parameters (as plist keys) and values supported by the API."
           (gptel--make-bedrock
            :name name
            :host host
-           :header nil           ; x-amz-security-token is set in curl-args if needed
```

Comment:
Why not set the bearer token in curl-args as well to keep the logic centralized? See https://github.com/karthink/gptel/pull/1007/files as an example

Reply:
Good idea 👍

I guess we could allow the API key to be set as either an elisp parameter, or as an environment variable?